CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ v0.22 + 1
 * Updated binary identification in CRLF filtering to avoid false positives in
   UTF-8 files.
 
+* LF -> CRLF filter now correctly honors SafeCrLf=true errors
+  * LF only files were accepted with AutoCrLf=true on CRLF platforms
```

Comment:
Please let's not use camelcase here.  For describing the actual character, please use "CR" or "LF".  For describing the configuration option, please use all lowercase.  "core.autocrlf", that sort of thing.